src/PhappCommandBase.php

```diff
@@ -101,6 +108,12 @@ protected function getPhappEnviromentVariables($directory = './') {
 
     $env_vars = [];
     foreach ($finder as $file) {
+      // Add env vars from subapp manifest.
+      $subappManifest = $this->getSubappManifest($directory);
```

Comment:
This seems to take care about subapps only, but it really should work with all apps, including main app. Not sure it does?